source/documentation/common/install/proc-Defining_allowed_CPU.adoc

```diff
@@ -0,0 +1,61 @@
+:_content-type: PROCEDURE
+[id="proc-proc-Defining_allowed_CPU_{context}"]
+= Defining allowed CPU types in Self-hosted Engine deployment
+
+[role="_abstract"]
+
+
+
+.Procedure
+
+. Create a file named deploy.json, and provide a CPU type name for the value `cpu_type_name` from the table shown below. For example, if the CPU type you want is *Secure Intel Nehalem Family*, then the `deploy.json` should look like the following:
++
+----
+[root@host ~]# cat deploy.json
+{
+	"he_cluster_cpu_type": "Secure Intel Nehalem Family"
+}
+
+----
+. Provide the deploy.json file to the `hosted-engine --deploy` process.
++
+----
+[root@host ~]# hosted-engine --deploy --ansible-extra-vars=@/root/deploy.json
+----
+
+[id="Allowed_CPU_types"]
+.Allowed CPU Types
+[options="header"]
+|===
+|CPU type name |CPU properties
```

Comment:
Same with the CPU properties.

Reply:
in this case, it's just a generic column title, not the name of a variable.